.github/workflows/ci.yml

```diff
@@ -33,5 +33,5 @@ jobs:
           kubernetes version: ${{ matrix.kubernetes }}
           driver: 'docker'
       - name: Run integration tests
-        run: mvn -B package -P no-unit-tests --file pom.xml
+        run: mvn -e -B package -P no-unit-tests --file pom.xml
```

Comment:
Do we need -e switch here?

Reply:
I think it's helpful when the tests fail because of an unexpected exception but it's definitely not needed.